### PR TITLE
Remove storybook

### LINKED ISSUE #459: Remove storybook and stories
Remove our dependencies on storybook, remove all stories. We never really ended up using it, and we're just using StyleGuide to handle it.

### PR DESCRIPTION
**Pull request by mofojed**
We never really used it.
Fixes #459

### CODE REVIEW

File: packages/grid/package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
   "devDependencies": {
     "@babel/cli": "^7.16.0",
     "@deephaven/tsconfig": "^0.11.0",
+    "@types/color-convert": "^2.0.0",
```
Comment (mattrunyon) [inline packages/grid/package.json:38]: Where did this come from? Also it shouldn't be in regular dependencies if this needs to stay (it's in both dev and regular right now)

**Review by mattrunyon — CHANGES_REQUESTED** [review]

File: packages/grid/package.json (hunk 1 above)
Comment (mofojed) [reply packages/grid/package.json:38]: I was getting a build error:
```
packages/grid/src/GridColorUtils.ts:1:21 - error TS7016: Could not find a declaration file for module 'color-convert'. '/home/bender/dev/deephaven/iris-oss/web-client-ui/main/node_modules/color-convert/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/color-convert` if it exists or add a new declaration (.d.ts) file containing `declare module 'color-convert';`

1 import convert from 'color-convert';
                      ~~~~~~~~~~~~~~~

packages/grid/src/GridColorUtils.ts:2:21 - error TS7016: Could not find a declaration file for module 'color-convert/conversions'. '/home/bender/dev/deephaven/iris-oss/web-client-ui/main/node_modules/color-convert/conversions.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/color-convert` if it exists or add a new declaration (.d.ts) file containing `declare module 'color-convert/conversions';`

2 import { HEX } from 'color-convert/conversions';
                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~

```

Will remove from dependencies

Comment (mofojed) [reply packages/grid/package.json:38]: It was required by `@storybooks/components`, so that's why it was getting installed before

**Review by mattrunyon — APPROVED** [review]
I didn't find any more references/files w/ story or stories

**Pull request merged**